community/pmc-tool/collector/gh.py

```diff
@@ -0,0 +1,18 @@
+import requests
```

Comment:
please add the Apache lincese header in this file.

please also add documentation to the file (e.g. `"""This file contains functionality to query data from Github"""`).